ldapauthenticator/tests/test_ldapauthenticator.py

```diff
@@ -100,3 +100,14 @@ async def test_ldap_auth_state_attributes(authenticator):
     )
     assert authorized["name"] == "fry"
     assert authorized["auth_state"] == {"employeeType": ["Delivery boy"]}
+
+async def test_ldap_auth_state_attributes2(authenticator):
+    authenticator.group_filter = "(cn=ship_crew)"
```

Comment:
Played with this value, with the Docker container (same used in CI), causing it to fail with different values. Also tested locally with a different docker image that contains a more restrict schema in OpenLDAP, allowing only `member` in the search.